src/main/kotlin/ai/digital/integration/server/deploy/internals/cluster/DeployDockerClusterHelper.kt

```diff
@@ -71,6 +77,14 @@ open class DeployDockerClusterHelper(val project: Project) {
         return "${worker.dockerImage}:${getClusterVersion()}"
     }
 
+    private fun getWorkerJdbcUrl(): String {
+        var workerJdbcUrl = "postgresql:5432/xld-db"
```

Comment:
This is already not correct, as it depends what database is used, and this config has to be configurable.